design/design.md

```diff
@@ -0,0 +1,414 @@
+the [paper](https://eprint.iacr.org/2023/1655) to get a more detailed overview
+of ALBA.
+
+### Applications of ALBA
+
+This cryptographic primitive has a number of applications, in the Cardano space
+and outside.
+
+Mithril is a system that allows SPOs to certify the current state of the Cardano
+blockchain, allowing, for example, nodes to get up-to-date significantly faster
+than by processing the whole chain of blocks.
+The core component of Mithril is a stake-based (weighted) threshold
+multisignature scheme, also called Mithril, that lets stakeholders collectively
+sign a message that convinces a verifier that nodes with more than some
+threshold of the participating stake signed this message.
+To faciliate fast node bootstrapping, this message is a commitment to the whole
+state of the ledger.
+ALBA can be used to construct a new weighted threshold multisignature scheme
+with a significantly better proof size which
+can allow Cardano nodes to process the chain of Mithril certificates faster.
+If a SNARK is used to compress Mithril certificates to a very small size,
+ALBA can reduce the overall proving time.
+
+Peras and Leios are new extensions of the Ouroboros consensus protocol used in
+Cardano.
+Ouroboros Peras aims to reduce the block settlement time by executing a byzantine fault
+tolerant agreement protocol to decide the next block when adversarial
+conditions are mild.
+The consensus participants certify the chosen block to allow bootstrapping
+nodes to follow the right chain.
+A natural method for this certification is to use a
+weighted threshold multisignature scheme constructed using decentralized ALBA.
+
+Ouroboros Leios aims to increase the throughput of the blockchain in terms of
+the number of transactions per second.
+It does so by decoupling the block payloads from the consensus mechanism which
+operates only on references to blocks.
+To make sure that the chosen block's payload is correct and available
+(which potentially could be not due to malicious behavior),
+all blocks in the input to the consensus protocol need to have
+a certificate of availability proving that at least one honest node has verified
+the block content and is available to distribute it.
+A natural solution to certify data availability would be to use a
+weighted threshold multisignature scheme constructed using decentralized ALBA.
+
+Cross-chain bridges are protocols allowing one blockchain to verify
+information on another blockchain.
+In essense this is the same problem that the Mithril solves with a potentially
+added constraint that on-chain verification must be computationally cheap.
+Whereas Mithril is specific to the Cardano blockchain, one can envision
+a similar protocol, powered by ALBA, for other cryptocurrencies including
+partner chains like Midnight.
+
+Applications of ALBA are not limited to Cardano and blockchains in general.
+For instance, the [paper](https://eprint.iacr.org/2023/1655) describes how
```

Comment:
It still gives me the same feeling, since we start the sentence with "For instance". 
However, it is not a big issue. The reader will eventually see which paper it is when they click on the link.